packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
Are you talking about the `...a`? It's not actually "dropped", it's optimized away to `arguments` accesses. I can add another test where it's deopted (and creates the `...a` array) if you'd like.